pcsc.c

```diff
@@ -1,8 +1,6 @@
 /*
  +----------------------------------------------------------------------+
- | PHP Version 5                                                        |
- +----------------------------------------------------------------------+
- | Copyright (c) 1997-2006 The PHP Group                                |
+ | Copyright (c) The PHP Group                                          |
```

Comment:
As in main php-src (better to removed than trying to maintain this outdated information)